.expeditor/config.yml

```diff
@@ -8,7 +8,6 @@ slack:
   notify_channel: chef-server-notify
 
 github:
-  delete_branch_on_merge: true
```

Comment:
We should keep this if that's what you want to happen. I'll make sure it gets fixed next week.

Reply:
That would be awesome! Thanks @tduffield!